src/h3lib/lib/h3Index.c

```diff
@@ -663,3 +645,118 @@ H3Index H3_EXPORT(geoToH3)(const GeoCoord* g, int res) {
     _geoToFaceIjk(g, res, &fijk);
     return _faceIjkToH3(&fijk, res);
 }
+
+/**
+ * Convert an H3Index to the FaceIJK address on a specified icosahedral face.
+ * @param h The H3Index.
+ * @param fijk The FaceIJK address, initialized with the desired face
+ *        and normalized base cell coordinates.
+ * @return Returns 1 if the possibility of overage exists, otherwise 0.
+ */
+int _h3ToFaceIjkWithInitializedFijk(H3Index h, FaceIJK* fijk) {
```

Comment:
nit: `s/_h3ToFaceIjkWithInitializedFijk/_h3ToFaceIJKWithInitializedFIJK` and same for others

Reply:
So that's how the original versions of these functions were named in H3IndexFat, and it makes a kind of sense to camelCase that way since it would be obvious to split as "h3 to face ijk with initialized fijk" whereas the other way would split as "h3 to face i j k with initialized f i j k".

Reply:
If that’s how it was, no need to bunch unrelated changes into this.  Comment rescinded!

I don’t buy the `Ijk` makes more sense than `IJK` argument, though :P